Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
 | Unit test | does OIP have unit tests for this entity; denoted by symbol, see [legend above](#legend) |
 | Issue | Link to corresponding open issue(s) |
 
-| Entity                | Covered              | Converter                  | Unit test             | Issue |
-|-----------------------|----------------------|----------------------------|-----------------------|-------|
-| `IfcCartesianPoint`   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
-| `IfcPointOnCurve`     | :x:                  | -                          | -                     | [#359](https://www.github.com/tumcms/Open-Infra-Platform/issues/359) |
+| Entity                                | Covered              | Converter                  | Unit test             | Issue |
+|---------------------------------------|----------------------|----------------------------|-----------------------|-------|
+| `IfcAdvancedBrep`                     | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcAdvancedBrepWithVoids`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcAlignmentCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcAnnotationFillArea`               | :x:                  | -                          | -                     | -     |
+| `IfcAxis1Placement`                   | :clock9:             | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2Placement2D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2Placement3D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2PlacementLinear`             | :x:                  | `PlacementConverter`       | -                     | -     |
+| `IfcBlock`                            | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBlossCurve`                       | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcBooleanClippingResult`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcBooleanResult`                    | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBoundedCurve`                     | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcBoundingBox`                      | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBoxedHalfSpace`                   | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcBSplineCurveWithKnots`            | :heavy_check_mark:   | `SplineConverter.`         | -                     | -     |
```

Comment:
I'm not sure that we really support this. @christophKaiser can you confirm?

Also, what's with the dot in SolidConverter?

Reply:
The function `SplineConverterT::convertIfcBSplineCurve` supports `IfcBSplineCurveWithKnots`, and `IfcRationalBSplineCurveWithKnots` as well.
However, until now I wasn't able to test this function because I didn't found an appropriate ifc testdata file. All files, which I have analysed, use an BSplineCurve as sub entity of an `IfcBSplineSurface` which isn't supported at the moment.

Reply:
Resolved 31a5bd4